coalahtml/_coalahtml/.coafile

```diff
@@ -0,0 +1,16 @@
+[js]
```

Comment:
please remove this file.  It is already covered by the file https://github.com/coala/coala-html/blob/master/.coafile .
I do appreciate that you needed it to generate the image, and so it was helpful for that purpose.  but it isnt needed in the repository

Reply:
And likewise remove everything in the commit message after `Closes #141`.